models/DDPOModel.js

```diff
@@ -2,11 +2,11 @@ const mongoose = require('mongoose');
 const { collatorDb } = require('../dbs');
 
 const DDPOSchema = new mongoose.Schema({
-  name: {
-    type: String, unique: true, minLength: 2, maxLength: 40, required: true,
-  }
+    name: {
```

Comment:
For readability the indentation for this project has been set as 2 spaces rather than a full 4 space tab. You'll need to set this in your workspace and apply it to this pr, so that we don't have large amounts of whitespace change in every future pr